cxx/structureTranslationModule/test/units/testStructureTranslationAgent.cpp

```diff
@@ -0,0 +1,208 @@
+/*
+ * This source file is part of an OSTIS project. For the latest info, see http://ostis.net
+ * Distributed under the MIT License
+ * (See accompanying file COPYING.MIT or copy at http://opensource.org/licenses/MIT)
+ */
+ 
+#include "scs_loader.hpp"
+
+#include "sc-agents-common/utils/AgentUtils.hpp"
+#include "sc-agents-common/utils/CommonUtils.hpp"
+#include "sc-agents-common/utils/IteratorUtils.hpp"
+
+#include "keynodes/TranslationKeynodes.hpp"
+
+#include "agent/StructureTranslationAgent.hpp"
+
+#include "translator/StructureTranslatorSet.hpp"
+#include "translator/StructureTranslator.hpp"
+#include "translator/NrelInLinkTranslator.hpp"
+#include "translator/NrelInQuasybinaryLinkTranslator.hpp"
+#include "translator/NrelInQuasybinaryNodeTranslator.hpp"
+#include "translator/NrelFromQuasybinaryNodeTranslator.hpp"
+#include "translator/FromConceptTranslator.hpp"
+#include "translator/NrelFromNodeTranslator.hpp"
+#include "translator/NrelFromQuasybinaryLinkTranslator.hpp"
+
+#include "sc-memory/kpm/sc_agent.hpp"
+#include "sc_test.hpp"
+
+#include <algorithm>
+
+using namespace structureTranslationModule;
+
+namespace StructureTranslationAgent
+{
+ScsLoader loader;
+std::string const & TEST_FILES_DIR_PATH = STRUCTURE_TRANSLATION_MODULE_TEST_SRC_PATH "/testStructures/StructureTranslation/";
+std::string const & TEST_QUESTION_NODE_ALIAS = "test_question_node";
+std::string const & TEST_STRUCTURE_ALIAS = "test_structure";
+int const WAIT_TIME = 3000;
+
+using StructureTranslationTest = ScMemoryTest;
+
+void initializeClasses()
+{
+  scAgentsCommon::CoreKeynodes::InitGlobal();
+  TranslationKeynodes::InitGlobal();
+  ScAgentInit(true);
+  SC_AGENT_REGISTER(structureTranslationModule::StructureTranslationAgent);
+}
+
+void deinitializeClasses()
+{
+  SC_AGENT_UNREGISTER(structureTranslationModule::StructureTranslationAgent);
+}
+
+void testTranslator(ScMemoryContext & context, auto & translator, std::string fileName, std::vector<std::string> answer_phrases)
```

Comment:
```suggestion
void testTranslator(ScMemoryContext & context, auto & translator, std::string fileName, std::vector<std::string> answerPhrases)
```
We don't use snake_case variable names, check in this file